app/views/kaui/refunds/_form.html.erb

```diff
@@ -158,8 +179,32 @@
         }
         recomputeRefundAmountAndValidateAmount();
         validateInvoiceItemAmount(id);
+        checkSelectAllCheckboxStatus();
     };
 
+    /*
+    * When clicking select all checkbox - select each items, disable amount and recompute total refund amount
+    */
+    var onClickInvoiceItemsSelectAll = function(event){
+        var isChecked = false;
+        if ($(this).is(':checked')) {
```

Comment:
as we have this function specifically for selectAll (as it is reflected in its name), I think we can directly check by id like:
`var isChecked = $('#selectAll').prop('checked');`

Reply:
We have changed `$(this).is(':checked')` to `$('#selectAll').prop('checked');` as `$('#selectAll')` make more sense here.